src/utils.jl

```diff
@@ -0,0 +1,49 @@
+StatsBase.describe(X::NullableVector) = StatsBase.describe(STDOUT, X)
+
+function StatsBase.describe(io::IO, X::NullableVector{<:Real})
+    if allnull(X)
+        println(io, " * All null * ") && return
```

Comment:
Fixing this now, but rather than correct this line I'm inclined to suggest removing this case? This is how it is in both of the tables packages but it's less informative than just using the base-case report, e.g.
```julia
julia> describe(NullableArray{Any}(5))
 * All null * 
```

isn't as informative as 
```julia
julia> describe(NullableArray{Any}(5))
Summary Stats:
Length:       5
Type:         Nullable{Any}
Unique:       1
Null:         5
% Null:       100.000000
```

another option being to still throw ` * All null * ` but do so in addition to the full report (either at the beginning or the end) like so
```julia
julia> describe(NullableArray{Any}(5))
Summary Stats:
Length:       5
Type:         Nullable{Any}
Unique:       1
Null:         5
% Null:       100.000000
 * All null *
```

although that last option is more code and doesn't say anything more than `% Null:       100.000000` could tell us.